app/models/post_custom_field.rb

```diff
@@ -17,7 +17,6 @@ class PostCustomField < ActiveRecord::Base
 #
 # Indexes
 #
-#  idx_post_custom_fields_akismet                (post_id) WHERE (((name)::text = 'AKISMET_STATE'::text) AND (value = 'needs_review'::text))
```

Comment:
Oh my... this was an index added to core. I'll need to drop this index and add it into akismet